core/tests/meson.build

```diff
@@ -7,7 +7,7 @@
 #
 
 # Note: this version of cmpfiles ignores line endings, which is better for platform independence
-cmpfiles = ['-c', 'import sys; a = open(sys.argv[1], \'r\').read(); b = open(sys.argv[2], \'r\').read(); exit(not (a==b))']
+cmpfiles = ['-c', 'import sys; a = open(sys.argv[1], \'r\').read(); b = open(sys.argv[2], \'r\').read(); sys.exit(not (a==b))']
```

Comment:
Unrelated ... but maybe due to my python version? In any case `sys.exit` is probably safer